covid19/index.html

```diff
@@ -107,14 +107,12 @@ <h2 data-i18n="[html]ENGAGE_FILE_TRANSFORMER"></h2>
                     </div>
                     <div class="step step2">
                         <div class="white-container">
-                            <div class="form-group">
-                                <label for="form-institution" data-i18n="INSTITUTION"></label>
+                            <div class="form-group" style="display: none;">
                                 <input
-                                    type="text"
+                                    type="hidden"
```

Comment:
Sorry, I should've made this more clear in the description. The file transformer will still be available outside of BB Assist as well. In those cases there will not be a _siteId_ parameter in the query string and we should ask the user where they are from.